### PR TITLE
feat: parameterize mev_boost and  mev_builder images

### PR DESCRIPTION
**Pull request by bharath-123**
Fixes: https://github.com/kurtosis-tech/eth2-package/issues/169

### CODE REVIEW

File: src/package_io/parse_input.star [hunk 1]
```diff
@@ -253,40 +258,24 @@ def enrich_mev_extra_params(parsed_arguments_dict, mev_prefix, mev_port, mev_typ
...
-				"--builder.genesis_validators_root={0}".format(package_io.GENESIS_VALIDATORS_ROOT_PLACEHOLDER),
-				"--miner.extradata=\"Illuminate Dmocratize Dstribute\"",
-				"--builder.algotype=greedy"
-				] + parsed_arguments_dict["mev_params"]["mev_builder_extra_args"],
-			"validator_extra_params": ["--builder-proposals"],
-			"builder_network_params": None
-		}
-
-		parsed_arguments_dict["participants"].append(mev_participant)
+	mev_url = "http://{0}{1}:{2}".format(mev_prefix, num_participants, mev_port)
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:261]: the if condition is necessary! we only do this when the mev is of type full not mock

File: src/package_io/parse_input.star [hunk 2]
```diff
@@ -219,6 +221,9 @@ def default_participant():
 def get_default_mev_params():
 	return {
 		"mev_relay_image": "flashbots/mev-boost-relay",
+		# TODO replace with actual when flashbots/builder is published
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:224]: can replace the TODO with "TODO replace with flashbots/builder when they publish an arm64 image" maybe tag the following issues

https://github.com/flashbots/builder/issues/107
https://github.com/flashbots/builder/issues/105

File: src/package_io/parse_input.star [hunk 3]
```diff
@@ -253,40 +258,24 @@ def enrich_mev_extra_params(parsed_arguments_dict, mev_prefix, mev_port, mev_typ
...
+		"el_client_type": "geth",
+		"el_client_image": parsed_arguments_dict["mev_params"]["mev_builder_image"],
+		"el_client_log_level":    "",
+		"cl_client_type":         "lighthouse",
+		# THIS overrides the beacon image
+		"cl_client_image":        "sigp/lighthouse",
+		"cl_client_log_level":    "",
+		"beacon_extra_params":    ["--builder={0}".format(mev_url), "--always-prepare-payload", "--prepare-payload-lookahead", "12000"],
+		# TODO(maybe) make parts of this more passable like the mev-relay-endpoint & forks
+		"el_extra_params": ["--builder",  "--builder.remote_relay_endpoint=http://mev-relay-api:9062", "--builder.beacon_endpoints=http://cl-{0}-lighthouse-geth:4000".format(num_participants+1), "--builder.bellatrix_fork_version=0x30000038", "--builder.genesis_fork_version=0x10000038", "--builder.genesis_validators_root={0}".format(package_io.GENESIS_VALIDATORS_ROOT_PLACEHOLDER), "--builder.algotype=greedy", "--miner.extradata=\"Illuminate Dmocratize Dstribute\""] + parsed_arguments_dict["mev_params"]["mev_builder_extra_args"],
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:274]: lets let this all be properly in different lines

**Review by h4ck3rk3y — CHANGES_REQUESTED** [review]

File: src/package_io/parse_input.star (hunk 3 above)
Comment (bharath-123) [reply src/package_io/parse_input.star:274]: cool i think my editor properly formatted it, will do it

File: src/package_io/parse_input.star (hunk 2 above)
Comment (barnabasbusa) [reply src/package_io/parse_input.star]: We now have a multiarch builder going for flashbots builder image: https://github.com/ethpandaops/eth-client-docker-image-builder/pull/80

**Pull request merged**